src/client/style.css

```diff
@@ -977,12 +977,12 @@ p#splash-text span {
 
 .other div#carshare .box.active {
     background-color: green;
-}
```

Comment:
Could you please try removing the whitespace in another commit and force push to this same PR?